java/ql/test/qlpack.yml

```diff
@@ -1,5 +1,5 @@
 name: codeql/java-tests
-version: 0.0.2
+groups: [java, test]
```

Comment:
No version here? I guess we don't need it since it won't be published.

Reply:
If we never run `pack create` on it, we really don't need a version at all.

Reply:
Maybe we should make sure it is an error to run `pack create` on a pack that doesn't have a version. Not sure what the behaviour is now.